packages/rspack/tests/Defaults.unittest.ts

```diff
@@ -445,99 +445,101 @@ describe("snapshots", () => {
 	);
 	test("no mode provided", { mode: undefined }, e =>
 		e.toMatchInlineSnapshot(`
-		- Expected
-		+ Received
+- Expected
```

Comment:
This indentation diff make the pr hard to review

Reply:
I don't know why `jest --updatesnapshot` will change its indent, how can I update snapshot correctly

Reply:
please address other issues, this diff is hard to review, but acceptable.